app/src/main/java/com/oracle/javafx/scenebuilder/app/preferences/PreferencesWindowController.java

```diff
@@ -52,6 +52,7 @@
 import static com.oracle.javafx.scenebuilder.app.preferences.PreferencesController.RECENT_ITEMS;
 import static com.oracle.javafx.scenebuilder.app.preferences.PreferencesController.RECENT_ITEMS_SIZE;
 import static com.oracle.javafx.scenebuilder.app.preferences.PreferencesController.TOOL_THEME;
+import static com.oracle.javafx.scenebuilder.app.preferences.PreferencesController.WILDCARD_IMPORT;
```

Comment:
Where is this defined?? I don't see `WILDCARD_IMPORT` added to `PreferencesController`.

Reply:
It is defined in the `PreferencesControllerBase` class ;)